### PR TITLE
feat(DTUPSTREAM-231): Add credit risk verification to expand the tagline to other countries

### PR DESCRIPTION
**Pull request by lucasfrotam**
### Description

- Adding the credit risk verification to use in Elmo Experiment because we will expand the 'Pay now or pay later' to the other countries which have some credit risk (UK, DE, FR, AU).

### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues

- Jira task: https://engineering.paypalcorp.com/jira/browse/DTUPSTREAM-231

### Dependent Changes 

- xobuyernodeserv PR: https://github.paypal.com/Checkout-R/xobuyernodeserv/pull/2454

❤️  Thank you!

### CODE REVIEW

**Review by erick2014 — APPROVED** [review]

**Review by wsbrunson — APPROVED** [review]

**Comment by lucasfrotam** [comment]
retest this please

File: server/service/personalization.jsx [hunk 1]
```diff
@@ -158,6 +162,7 @@ export async function resolvePersonalization(req : ExpressRequest, gqlBatch : Gr
     const ip = req.ip;
     const cookies = req.get('cookie') || '';
     const userAgent = req.get('user-agent') || '';
+    const creditRiskVerified = fundingEligibility && fundingEligibility.credit?.eligible;
```
Comment (mstuart) [inline server/service/personalization.jsx:165]: Can we use optional chaining in this project? TIL something :)

**Comment by mstuart** [comment]
Thanks for calling out the xobuyernodeserv dependency. I assume that's going out next week? To be safe, let's wait to merge this until after that goes out.

**Review by mnicpt — APPROVED** [review]

File: server/service/personalization.jsx (hunk 1 above)
Comment (westeezy) [reply server/service/personalization.jsx:165]: `babel-register` is added to smartcomponents during build so in theory this should be fine. that being said however I am bit nervous about adding it given I've been bitten with these assumptions before. @lucasfrotam were you testing this through smartcomponents?

Comment (lucasfrotam) [reply server/service/personalization.jsx:165]: hi guys, sorry for the delay, but yes, I tested the entire flow going through xobuyer, smartcomponents and rendering the PP button on the page

Comment (lucasfrotam) [reply server/service/personalization.jsx:165]: hi guys, sorry for the delay, but yes, I tested the entire flow going through xobuyer, smartcomponents and rendering the PP button on the page

**Pull request merged**